src/tunnel.ts

```diff
@@ -13,8 +13,6 @@ interface TunnelConfig extends ConnectConfig {
   dstHost: string;
   dstPort: number;
   sshPort: number;
-  localHost: string;
-  localPort: number;
```

Comment:
Once I upgraded to ssh v0.8 the tunnel started to fail, that was happening because they added `localPort` option for the `client.connect` call, and we were passing the whole config object to that method which caused the new property to be used and broke our tunnel implementation. Turns out we don't need these props anyway, the db adapter wasn't really using it, the adapter gets the host and port of the forward connection from the server address.